tests/handlers/api/controller/test_node.py

```diff
@@ -108,6 +108,59 @@ def test_get_node(http_controller, tmpdir, project, compute):
     assert response.status == 200
     assert response.json["name"] == "test"
 
+def test_get_nodes_filtered_by_type(http_controller, tmpdir, project, compute):
+    response = MagicMock()
+    response.json = {"console": 2048}
+    compute.post = AsyncioMagicMock(return_value=response)
+
+    response = http_controller.post("/projects/{}/nodes".format(project.id), {
+        "name": "test",
+        "node_type": "vpcs",
+        "compute_id": "example.com",
+        "properties": {
+                "startup_script": "echo test"
+        }
+    })
+    http_controller.post("/projects/{}/nodes".format(project.id), {
+        "name": "test2",
+        "node_type": "ethernet_switch",
+        "compute_id": "example.com",
+        "properties": {
+                "startup_script": "echo test"
+        }
+    })
+    response = http_controller.get("/projects/{}/nodes?node_type=vpcs".format(project.id), example=True)
+    assert response.status == 200
+    assert len(response.json) == 1
```

Comment:
After running tests I get the following error
E       AssertionError: assert 500 == 200
E        +  where 500 = <ClientResponse(http://127.0.0.1:52909/v2/projects/7a5443b7-7a2b-4a39-8c9d-1797c1243468/nodes?status=started) [500 Int...NS3/2.3.0dev1', 'Content-Type': 'application/json', 'Content-Length': '763', 'Date': 'Fri, 03 Apr 2020 13:45:34 GMT')>\n.status